### ISSUE TITLE
Add a range for numbers used in a quiz

### ISSUE THREAD

**Issue by Dasert75** [h5p/h5p-arithmetic-quiz#46]
Hey! Is it possible to add a field that can limit the range of numbers used? For example 1-10, 1-30? This is useful for the elementary school.

**Comment by erenmustafaozdal** [comment]
Yes, definitely. I am an primary school teacher. 1st grade students need to addition between 1-20. Up to 100 in 2nd grade...

If we can set these limits, it will be very useful for primary school teachers.

**Comment by kurnias** [comment]
How can we add operations for negative numbers for addition and substraction from -100 to 100?
